MAUI/DataGrid/conditional-styling.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: Conditional Styling in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about Conditional Styling support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements and more.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Conditional Styling in .NET MAUI DataGrid (SfDataGrid)
+The [SfDataGrid] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.html) allows to customize the style of the individual cells and rows based on the requirements. 
+
+## Conditional row style
+The data rows can be customized conditionally by writing the style with converter for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) control. Its BindingContext is underlying datasource object.
```

Comment:
The data rows can be customized conditionally by writing the style with a converter for the [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) control. Its BindingContext is the underlying datasource object.